WooCommerce/Classes/ViewRelated/Products/AI/ProductDescriptionGenerationViewModel.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import Yosemite
+
+/// View model for `ProductDescriptionGenerationView`.
+final class ProductDescriptionGenerationViewModel: ObservableObject {
+    /// Product name, editable.
+    @Published var name: String
```

Comment:
Good question! Since the merchant can come to the flow without a product name, but there is little value to generate a product description without a name (not knowing what the product is), we're allowing the merchant to edit the product name here. We'll certainly keep iterating on the UX and maybe the product name will be required before this CTA in the future.